empathylink.lic

```diff
@@ -0,0 +1,130 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#empathylink
+=end
+
+custom_require.call(%w[common drinfomon equipmanager])
+
+class Empathylink
+  def initialize
+    if UserVars.empathylink.nil?
+      UserVars.empathylink=Hash.new
+    end
+    populate_noheal_list
```

Comment:
What is this? I couldn't find it in the repository. It's going to raise an exception.

Reply:
populate_noheal_list is a function just down from there, line 28.